js/src/data/selectors.js

```diff
@@ -68,15 +68,26 @@ export const getExistingGoogleAdsAccounts = ( state ) => {
 	return state.mc.accounts.existing_ads;
 };
 
-const mockPhoneNumber = Math.random() > 0.5 ? '+12133734253' : '';
-export const getGoogleMCPhoneNumber = () => {
-	// TODO: [lite-contact-info] integrate with API
-	return {
-		id: '123456789',
-		phone_number: mockPhoneNumber,
-	};
+export const getGoogleMCContactInformation = ( state ) => {
+	return state.mc.contact;
 };
 
+// Create another selector to separate the `hasFinishedResolution` state with `getGoogleMCContactInformation`.
+export const getGoogleMCPhoneNumber = createRegistrySelector(
```

Comment:
> Are there any issues with that approach?

Since the store address and GMC phone number are provided by the same API endpoint, and it needs to re-fetch the address during editing. To achieve all of the following targets, I finally ended up with this solution.

- Only allow the `<StoreAddressCard>` enter the loading state after clicking the refresh button.
- Don't reset the phone number user entered/changed after clicking the refresh button.
- Don't issue duplicated API requests to load phone number and address separately.

Here is a GIF to show UI issues if we use `getGoogleMCContactInformation` for getting phone number:

![Kapture 2021-08-03 at 14 24 50](https://user-images.githubusercontent.com/17420811/127968079-7191dc86-e460-4e98-8d49-20f268c97dd9.gif)

Reply:
Wow! Thank you, especially the animated gif! 😀 👍 

(I'm thinking it may be better to have separate APIs for phone number and store address - one reason being phone number is stored in Google and store address is in the local WC site - but anyway I guess we can keep that out of this PR 😄 )